lib/controllers/include/ActuatorController.h

```diff
@@ -20,18 +27,28 @@ class ActuatorController {
   virtual void overrideDegPerSecChecks() final;
   virtual void reinstateDegPerSecChecks() final;
 
-  /* functions to be overriden by children */
-  virtual bool update()                                     = 0;
-  virtual void stop()                                       = 0;
-  virtual void reset()                                      = 0;
-  virtual bool reportAngleDeg(float &angle)                 = 0;
-  virtual bool reportAngularVelocityDegPerSec(float &speed) = 0;
+  virtual bool reportAngleDeg(float &angle) final;
+  virtual bool reportAngularVelocityDegPerSec(float &speed) final;
 
+  /* functions to be overriden by children */
+  virtual bool update()                                            = 0;
+  virtual void reset()                                             = 0;
   virtual std::optional<std::reference_wrapper<PID::PID>> getPID() = 0;
+  virtual void stop()                                              = 0;
 
  protected:
+  virtual bool shouldStop() final;
+
   std::atomic<float> m_setpoint             = {0};
-  std::atomic<bool> m_ignoreCurrentChecks   = {false};
+  std::atomic<bool> m_ignoreCurrentChecks   = {true};  // TODO: Enable once Current Sensor Driver exists
   std::atomic<bool> m_ignoreDegPerSecChecks = {false};
```

Comment:
I think the current check / rpm check initial values should be user configurable